text/0033-private-network-guide.md

```diff
@@ -52,18 +50,15 @@ Here is a table summarizing the minimum requirements for each type of instance:
 |---  | ---:  | ---: | ---: | ---:     | --- |
 |Full node         | 3 | 4 | 8 GB | 4 GB | hathornetwork/hathor-core
 |Wallet            | 1 | 2 | 2 GB | 100 MB | hathornetwork/hathor-wallet-headless
-|Miner             | 2 | 2 | 1 GB | 100 MB | hathornetwork/cpuminer
 |Tx Mining Service | 1 | 2 | 1 GB | 100 MB | hathornetwork/tx-mining-service
 
-
 # Creating a new network
 
-The following steps must be executed to create a new network:
+The following steps should be executed to create a new network:
 
 1. Generate the configuration of the new network.
 2. Launch the full nodes and build the new p2p network.
 3. Launch the tx-mining-service to be able to send transactions.
-4. Launch the miner connected to the tx-mining-service.
```

Comment:
This step still exists in the text

Reply:
OK: change made:
Readded the step as suggested.